webapi/src/test/scala/org/knora/webapi/util/DateUtilSpec.scala

```diff
@@ -52,6 +54,54 @@ class DateUtilSpec extends WordSpec with Matchers {
             reverseConvertedBundesbriefDateValueV1 should be(bundesbriefDateValueV1)
         }
 
+        "convert an era date string with year precision to a Java GregorianCalendar BC" in {
+            val dateRange: DateRange = DateUtilV1.dateString2DateRange("50-02-28 BC", KnoraCalendarV1.GREGORIAN)
+            dateRange.start.get(Calendar.ERA) should be(GregorianCalendar.BC)
+        }
+
+        "convert an era date string with year precision to a Java GregorianCalendar AD" in {
+            val dateRange: DateRange = DateUtilV1.dateString2DateRange("50-02-28 AD", KnoraCalendarV1.GREGORIAN)
+            dateRange.start.get(Calendar.ERA) should be(GregorianCalendar.AD)
+        }
+        "convert an era date string with year precision to a Java GregorianCalendar BCE" in {
+            val dateRange: DateRange = DateUtilV1.dateString2DateRange("50-02-28 BCE", KnoraCalendarV1.GREGORIAN)
+            dateRange.start.get(Calendar.ERA) should be(GregorianCalendar.BC)
+        }
+        "convert an era date string with just year precision to a Java GregorianCalendar BCE" in {
+            val dateRange: DateRange = DateUtilV1.dateString2DateRange("50 BCE", KnoraCalendarV1.GREGORIAN)
+            dateRange.start.get(Calendar.ERA) should be(GregorianCalendar.BC)
+        }
+        "convert an era date string with just year/month precision to a Java GregorianCalendar BCE" in {
+            val dateRange: DateRange = DateUtilV1.dateString2DateRange("50-02 BCE", KnoraCalendarV1.GREGORIAN)
+            dateRange.start.get(Calendar.ERA) should be(GregorianCalendar.BC)
+        }
+        "convert an era date string with year precision to a Java GregorianCalendar CE" in {
+            val dateRange: DateRange = DateUtilV1.dateString2DateRange("50-02-28 CE", KnoraCalendarV1.GREGORIAN)
+            dateRange.start.get(Calendar.ERA) should be(GregorianCalendar.AD)
+        }
+        "convert an era date string with julian calendar with year precision to a Java GregorianCalendar BC" in {
+            val dateRange: DateRange = DateUtilV1.dateString2DateRange("50-02-28 AD", KnoraCalendarV1.JULIAN)
+            dateRange.start.get(Calendar.ERA) should be(GregorianCalendar.AD)
+        }
+
+        "convert a date in YYYY-MM-DD Era format, in the Julian calendar, into a Julian day count" in {
+            val bundesbriefDateValueV1 = DateValueV1(
+                dateval1 = "4713-01-01 BC",
```

Comment:
Swiss people might not be happy if you say that the Bundesbrief was in 4713 BC. :) https://de.wikipedia.org/wiki/Bundesbrief_von_1291